### DISCUSSION TITLE
I am getting this error often from the AI generation "Unknown Error: Unexpected token in JSON at position O"

### DISCUSSION THREAD

**Discussion by adamsonwalter** [webstudio-is/webstudio-community#136] [Q&A]
Unknown Error: Unexpected token in JSON at position O

Any clues here, see screenshot?

If I click on the first prompt, I get this error every time when executing it "Create a hero section with a heading, subheading in white, CTA button and an image of a mountain in the background,  add light blur to the image"

It doesn't happen with the second suggested prompt.

![Screenshot 2024-04-15 at 3 44 16 PM](https://github.com/webstudio-is/webstudio-community/assets/147778010/f9c60752-803d-4316-a61f-16113d1f1da6)

**Comment by kof** [comment]
I just tried and it worked, need to see some repro, could be unstable responses from openai

**Reply by kof** [reply]
Got it reproduced a second time around ... seems something became unstable

**Comment by kof** [comment]
Was able to get to the bottom of this, here is an isssue if you want to follow for updates https://github.com/webstudio-is/webstudio/issues/3174

**Comment by adamsonwalter** [comment]
Hi Oleg,

Now getting this
[image: Screenshot 2024-04-19 at 8.13.45 PM.jpg]

**Reply by kof** [reply]
screenshot didn't get uploaded, I bet you clicked submit too fast

**Comment by adamsonwalter** [comment]
[image: Screenshot 2024-04-19 at 8.13.45 PM.jpg]

**Reply by kof** [reply]
still no luck ...
